CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 ### Remove 'loading' screen when user has made an invalid query on job list page so the user can make changes.
 
+### Made behavior of message, log and execution icons consistent across all contexts in the job details page.
+
+Also fixed bug where log file links were being treated like directories.
```

Comment:
It may be more helpful to describe the user impact of this fix - "no more 404s" - instead of the underlying cause

Reply:
Changed to hopefully be clearer.